services.py

```diff
@@ -88,42 +76,80 @@ def get_param_from_config(self, config_path: str, param_name: str):
             self.logger.error(f'Config file not found: {config_path}')
             return None
 
-    def get_lat_lon(self):
-        g = geocoder.ip('me')
-        return g.latlng
+    def formatDegreesMinutes(self, coordinates, digits):
+    
+        parts = coordinates.split(".")
+
+        if (len(parts) != 2):
+            return coordinates
+
+        if (digits > 3 or digits < 2):
+            return coordinates
+        
+        left = parts[0]
+        right = parts[1]
+        degrees = str(left[:digits])
+        minutes = str(right[:3])
```

Comment:
What is contained in `right[0:3]` that you are removing it? You probably just need to write a docstring for this function giving some valid input examples

Reply:
The input is someting like this `2024-12-5.log`. I am removing the `.log`

Reply:
nope, this one is the different, it is about GPS parsing, not date parsing.